src/main/scala/org/broadinstitute/dsde/workbench/sam/api/SwaggerRoutes.scala

```diff
@@ -19,7 +19,7 @@ import scala.util.{Failure, Success, Try}
   * Created by dvoet on 7/18/17.
   */
 trait SwaggerRoutes extends LazyLogging {
-  private val swaggerUiPath = "META-INF/resources/webjars/swagger-ui/4.1.2"
+  private val swaggerUiPath = "META-INF/resources/webjars/swagger-ui/4.5.2"
```

Comment:
do we need to update this to 4.6.1 now?
```suggestion
  private val swaggerUiPath = "META-INF/resources/webjars/swagger-ui/4.6.1"
```

Reply:
shoot, and maybe also using `swagger-ui-dist`
```suggestion
  private val swaggerUiPath = "META-INF/resources/webjars/swagger-ui-dist/4.6.1"
```

Reply:
Oh, I did that but didn't push. Yes we do.